woocommerce/woocommerce
# Add block related assets entry points to product editor build

### All Submissions:






### Changes proposed in this Pull Request:

This PR creates a build process for blocks that extracts and copies assets around blocks for use in a later PR.  These assets will be required for individual block registration and third party use.

Closes (partially) #37241 .



### How to test the changes in this Pull Request:




1. From your terminal, run `pnpm --filter=@woocommerce/product-editor run build`
2. Check that a `build/blocks/` directory is created with the current (4) blocks
3. `section` and `tab` directories should be created with `editor.css` and `editor-rtl.css` files
4. Run `pnpm --filter=woocommerce/client/admin run build`
5. Check that the `woocommerce/assets/client/admin` directory contains a `product-editor/blocks` directory with all the previously built assets.



### Other information:




### FOR PR REVIEWER ONLY:
